functions/Install-DbaMaintenanceSolution.ps1

```diff
@@ -312,27 +314,25 @@ function Install-DbaMaintenanceSolution {
             $db = $server.Databases[$Database]
 
             if ($Solution -notcontains 'All') {
-                $required = @('CommandExecute.sql')
-            }
-
-            if ($LogToTable) {
-                $required += 'CommandLog.sql'
+                $required = @('CommandLog.sql', 'Queue.sql', 'QueueDatabase.sql', 'CommandExecute.sql') # Make all support procedures / tables required.
```

Comment:
@potatoqualitee FYI, I checked this as a breaking change PR because it is. This change specifically changes the functionality of the command and all of these objects are now going to be installed with the user-specified parameter `-LogToTable` is provided or not.

Reply:
We never documented it as such in the help, but still considered a breaking change in that the functionality no longer is the same.

Reply:
That seems unnecessary though, eh? I'm going to update it to only add those if log to table is specified